app/src/components/modal/modal_lock_yo_tokens/index.tsx

```diff
@@ -54,22 +46,23 @@ const DataRow = styled.div`
   width: 100%;
   line-height: ${props => props.theme.fonts.defaultLineHeight};
 `
+
+const LightDataItem = styled(TYPE.bodyRegular)<{ color: keyof Colors }>`
+  color: ${props => props.theme.text2};
+`
+const DarkDataItem = styled(TYPE.bodyMedium)<{ color: keyof Colors }>`
```

Comment:
Wondering whats the use of this <{ color: keyof Colors }> since you are not passing it inline but directly on the styled. It can be deleted....